bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -104,6 +102,8 @@ pub mod pallet {
 		#[cfg(feature = "runtime-benchmarks")]
 		type Helper: BenchmarkHelper<Self>;
 		type WeightInfo: WeightInfo;
+
+		type AccountToLocation: for<'a> TryConvert<&'a Self::AccountId, Location>;
```

Comment:
Why do you prefer this to the Location trait specification [here](https://github.com/vgeddes/polkadot-sdk/pull/3/files#diff-bfe761b371a553619edf5be2dd792c45bd14f1a916e51b77f5b77b9395d92cdaL194)?

Reply:
I see your answer here: https://github.com/paritytech/polkadot-sdk/pull/7402#discussion_r1941218780 Agreed!

Reply:
Converting an account should be infallible if the runtime is configured correctly. So rather use this trait: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_frame/traits/trait.Convert.html

Also using `xcm_builder::AccountAliasesToLocation` is incorrect as it always inserts the network id into the location, which we don't need.

Reply:
I prefer `TryConvert`, which can handle the error case.

> using xcm_builder::AccountAliasesToLocation is incorrect as it always inserts the network id into the location, which we don't need.

Just curious why we don't need it. Is it possible we send the reward to an address on Kusama?  

Maybe it is a bit over-engineering but I think adding the network id is not bad?